### PR TITLE
Reorganise how tos

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Reorganise TSDB How To documents into a more logical order.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

**Comment by Loquacity** [comment]
image

File: timescaledb/how-to-guides/page-index/page-index.js [hunk 1]
```diff
@@ -184,92 +241,167 @@ module.exports = [
...
+            keywords: ["schema", "hypertables", "TimescaleDB"],
+            excerpt: "Change the schema of a hypertable",
           },
           {
-            title: "Multi-node maintenance",
-            href: "multinode-maintenance",
-            tags: ["multi-node", "maintain", "distributed", "nodes", "manage"],
-            keywords: ["multi-node", "maintain"],
-            excerpt: "Maintain a multi-node environment",
+            title: "Indexing",
```
Comment (Rajakavitha1) [inline timescaledb/how-to-guides/page-index/page-index.js:306]: How about Indexes?

File: timescaledb/how-to-guides/page-index/page-index.js [hunk 2]
```diff
@@ -430,30 +522,11 @@ module.exports = [
...
-          },
-        ],
+        title: "Alerting",
+        href: "alerting",
+        tags: ["alert", "configure", "timescaledb"],
+        keywords: ["alert", "configure", "timescaledb"],
+        excerpt: "Configure alerting within TimescaleDB",
       },
       {
         title: "Data Tiering",
```
Comment (Rajakavitha1) [inline timescaledb/how-to-guides/page-index/page-index.js:532]: How about Data Tiers?

**Comment by Rajakavitha1** [comment]
LGTM from me :)

**Review by Rajakavitha1 — APPROVED** [review]

File: timescaledb/how-to-guides/page-index/page-index.js [hunk 3]
```diff
@@ -430,30 +522,11 @@ module.exports = [
...
-          },
-          {
-            title: "Data retention with continuous aggregates",
-            href: "data-retention-with-continuous-aggregates",
-            tags: ["retention", "caggs", "data", "timescaledb"],
-            keywords: ["retention", "data", "caggs", "TimescaleDB"],
-            excerpt: "Using data retention policies with continuous aggregates",
-          },
-        ],
+        title: "Alerting",
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply timescaledb/how-to-guides/page-index/page-index.js:525]: Probably, but let's leave it for when we update each of these. I'd like to keep this PR to the reorganisation only, if we can.

**Review by charislam — APPROVED** [review]

**Comment by ryanbooz** [comment]
My big question is, "why this order?". For instance, I would assume/think that Continuous Aggregates would be higher based on the overall question frequency, but maybe I'm wrong there.

**Comment by Loquacity** [comment]
> My big question is, "why this order?". For instance, I would assume/think that Continuous Aggregates would be higher based on the overall question frequency, but maybe I'm wrong there.

I was going more for the order that you would naturally do things in, rather than on how 'popular' a topic might be. Happy to rearrange, though.

**Pull request merged**